pallets/committee/src/lib.rs

```diff
@@ -254,6 +247,23 @@ decl_module! {
             Self::deposit_event(RawEvent::VoteThresholdUpdated(current_did, n, d));
         }
 
+        /// Changes the release coordinator.
+        ///
+        /// # Arguments
+        /// * `id` - The DID of the new release coordinator.
+        ///
+        /// # Errors
+        /// * `MemberNotFound`, If the new coordinator `id` is not part of the committee.
+        #[weight = (500_000, Operational, Pays::Yes)]
+        pub fn set_release_coordinator(origin, id: IdentityId) {
+            T::CommitteeOrigin::ensure_origin(origin)?;
+            ensure!(Self::members().contains(&id), Error::<T, I>::MemberNotFound);
+            <ReleaseCoordinator<I>>::put(id);
+            let current_did = Context::current_identity::<Identity<T>>()
+                .unwrap_or(SystematicIssuers::Committee.as_id());
```

Comment:
Since this function can only be called by committee origin, we can directly set `current_did = SystematicIssuers::Committee.as_id()`